.gitignore

```diff
@@ -14,4 +14,4 @@
 heaptrack.*.gz
 3rdparty/wigner/fastwigxj/gen/
 3rdparty/wigner/fastwigxj/obj/
-
+cmake-build-debug
```

Comment:
Is this intended?

Reply:
Yes, that was on purpose. That folder showed up for me and it should not be committed.  Missed to comment on it.